sample-data/src/main/java/com/kodemon/sampledata/SampleDataLoadingFacadeImpl.java

```diff
@@ -16,8 +24,35 @@
     @Inject
```

Comment:
We will need to refactor this to constructor DI. But don't do it here. We will do as soon as Matej merges his changes.